set.mm

```diff
@@ -674802,6 +674806,831 @@ fixed reference functional determined by this vector (corresponding to
+      VWSVWQUWQVWSRVUIOVGVWQUWQUWJVUIROVUJXNUWQVUIVUGROVULXNXLXRUWQVWSVWOUWIVJZ
+      VWPUWQRVEVIZUWIVJZOVGZRVXAOVGZUWIVJZVKZVWSVWTVKZVEUWOUWHVXAUWHVKZVXCVWSVX
+      EVWTVXHVXBUWJROVXAUWHUWIYJXNVXHVXDVWOUWIVXAUWHROYOVRYKAVXFVEUWOVMZUWPAVXG
+      CUWOVMZVXIARPJVHVXJVBACEOJKMNROPTSUCAUYCUXHALUVPVNZUBEVNZUXBEVNZXBUXHAVXK
+      VXLVXMALAUXKLWCVNUXLLQUDUVQWKXOAVUBVXLVUCELQUBUFUDUEUVTWKZAVUBUYSVXMVUCVU
+      DFELQUWSUXIUDUJUXJUEUVRWQZYLEILUBUXBUEUNUWAWKAPUXCEUXFAUTVQUWBYMZUYDYNWRV
+      XGVXFCVEUWOUWHVXAVKZVWSVXCVWTVXEVXQUWJVXBROUWHVXAUWIYJXNVXQVWOVXDUWIUWHVX
+      AROYOVRYKUWCYDWEAUWPUWDUWEUWQVWTVWOUXDVJZVWPUWQVWOUWIUXDUXGVTUWQUXHVXRVWP
+      VKUWQUXILUWTIQEUBUXBSVWOUWSVWOUMUDUXJUEUXMUWQVWOUXNUXIUWQUXNOTRUWHUXOUIUX
+      QARWLVNUWPARUYDWSWEUYNWTUYQXCZUWQVXLVWOUBSVJVJVWOVKZAVXLUWPVXNWEUWQVXLVXT
+      UWQUXILQESUBVWOUMUFUXJUDUEUXMVXSXDXKYPUWQVXMVWOUXBSVJVJUWSVKZAVXMUWPVXOWE
+      UWQVXMVYAUWQFUXILQESUWSVWOUMUDUXJUJUEUXMVUEVXSXIXKYPUNVUFXJXKXLXLXLXLUWQV
+      WOVWMUWSVWBUWTUWQRVWKUWHOUWQVWKRVWRXRYIVWJYEYQUWQVWMVWAVWBUWTUWQVVOVVQUXR
+      UWHVVKWAVJZVNZXBVWMVWAVKVVPUWQVVQUXRVYCVVRUYGUWQUWHUXIVYBVUKVVMYRYLVYBOVV
+      KHUWGUWHVYBWBZVVNYSWQYIYTUWQVVOVVQUXRVUGVYBVNZXBVWLVVHVKVVPUWQVVQUXRVYEVV
+      RUYGUWQVUGUXIVYBUWQUXIUWTQUWHUWSUXJVUFVVEVUKVUEXPZVVMYRYLVYBOVVKHUWGVUGVY
+      DVVNYSWQYTXLZUWQVUSVVHUWQBVUHVUNVVHUXIVUOUXIVVIVUMVUHVKZVUNVVHVKUWQVYHVUM
+      VUHHOVYHXSXNXTUWQUXIOVVKUWGVUGVVMVVNVVPUYGVYFWTZUWQVVGVVHUXIVYGVVSYCYBXRY
+      FVRUWQVVAVUHUXIVNVUTVUHVKVVDVYIUXIUXIVUHVUOXQWQYFXRYQXLXRUWFACEOJKMNUWGOP
+      TSUCVXPUYFYNYM $.
+  $}
+
+  ${
+    $d .^ e f y $.  $d .^ y z $.  $d B e f $.  $d E e f y $.  $d E y z $.
+    $d F e f y $.  $d F y z $.  $d G e f y $.  $d G y z $.  $d O e f y $.
+    $d O y z $.  $d R e f y $.  $d R l y $.  $d R y z $.  $d V e f y $.
+    $d V l y $.  $d V y z $.  $d W e f y $.  $d l ph y $.
+    aks6d1c1p4.1 $e |- .~ = { <. e , f >. | ( e e. NN /\ f e. B /\
+    A. y e. ( V PrimRoots R )
+     ( e .^ ( ( O ` f ) ` y ) ) = ( ( O ` f ) ` ( e .^ y ) ) ) } $.
+    aks6d1c1p4.2 $e |- S = ( Poly1 ` K ) $.
+    aks6d1c1p4.3 $e |- B = ( Base ` S ) $.
+    aks6d1c1p4.4 $e |- X = ( var1 ` K ) $.
+    aks6d1c1p4.5 $e |- W = ( mulGrp ` S ) $.
+    aks6d1c1p4.6 $e |- V = ( mulGrp ` K ) $.
+    aks6d1c1p4.7 $e |- .^ = ( .g ` V ) $.
+    aks6d1c1p4.8 $e |- C = ( algSc ` S ) $.
+    aks6d1c1p4.9 $e |- D = ( .g ` W ) $.
+    aks6d1c1p4.10 $e |- P = ( chr ` K ) $.
+    aks6d1c1p4.11 $e |- O = ( eval1 ` K ) $.
+    aks6d1c1p4.12 $e |- .+ = ( +g ` S ) $.
+    aks6d1c1p4.13 $e |- ( ph -> K e. Field ) $.
+    aks6d1c1p4.14 $e |- ( ph -> P e. Prime ) $.
+    aks6d1c1p4.15 $e |- ( ph -> R e. NN ) $.
+    aks6d1c1p4.16 $e |- ( ph -> ( N gcd R ) = 1 ) $.
+    aks6d1c1p4.17 $e |- ( ph -> P || N ) $.
+    aks6d1c1p4.18 $e |- ( ph -> E .~ F ) $.
+    aks6d1c1p4.19 $e |- ( ph -> E .~ G ) $.
+    $( The product of polynomials is introspective, (Contributed by metakunt,
```

Comment:
```suggestion
    $( The product of polynomials is introspective. (Contributed by metakunt,
```

similar to above, looks like addition. (Also comma -> period)

affects aks6d1c1p5